vike/client/client-routing-runtime/prefetch/getPrefetchSettings.ts

```diff
@@ -109,3 +152,44 @@ function getPrefetchAttribute(linkTag: HTMLElement): PrefetchStaticAssets | null
 
   assert(false)
 }
+
+function getPrefetchPageContextAttribute(linkTag: HTMLElement): PrefetchPageContext | null {
+  const whenAttr = linkTag.getAttribute('data-prefetch-page-context-when')
```

Comment:
How about `data-prefetch-page-context` instead of `data-prefetch-page-context-when`?